### PR TITLE
Feature/36717 - Styles in 

### PR DESCRIPTION
**Pull request by kjugi**
Repair nav styles
Add dashboard table modifier
Add list styles to dahsboard - it tabs in my orders in fact

DEV-36717

### CODE REVIEW

File: components/03-modules/dashboard/nav/_nav.scss [hunk 1]
```diff
@@ -77,13 +77,19 @@ $dashboard-nav-margin : 15px !default;
...
+        margin-left: -$dashboard-nav-margin;
         border-left: 2px solid $red;
         color: $red;
-        position: relative;
 
         &:hover {
             background-color: initial;
         }
+
+        & > a {
```
Comment (kruchy8) [inline components/03-modules/dashboard/nav/_nav.scss]: Is this selector necessary ? Can we use some class for example ?

File: components/03-modules/dashboard/nav/_nav.scss [hunk 2]
```diff
@@ -77,13 +77,19 @@ $dashboard-nav-margin : 15px !default;
 
     // This class is changing by Magento
     .current {
+        position: relative;
+        padding: $dashboard-nav-padding $dashboard-nav-padding $dashboard-nav-padding ($dashboard-nav-padding * 2);
+        margin-left: -$dashboard-nav-margin;
         border-left: 2px solid $red;
```
Comment (kruchy8) [inline components/03-modules/dashboard/nav/_nav.scss]: I think we might use variable here.

File: components/03-modules/dashboard/table/_table.scss [hunk 3]
```diff
@@ -3,6 +3,16 @@ $dashboard-table-margin: 30px !default;
 .dashboard-table {
     overflow: hidden;
 
+    &--tab {
+        padding: 10px;
```
Comment (kruchy8) [inline components/03-modules/dashboard/table/_table.scss]: var

File: components/03-modules/dashboard/table/_table.scss [hunk 4]
```diff
@@ -3,6 +3,16 @@ $dashboard-table-margin: 30px !default;
...
+        margin-bottom: $dashboard-table-margin;
+        border: 1px solid $gray;
```
Comment (kruchy8) [inline components/03-modules/dashboard/table/_table.scss]: var

File: components/04-views/dashboard/default/_dashboard.scss [hunk 5]
```diff
@@ -118,4 +118,50 @@ $dashboard-padding: 15px !default;
...
+                    content: "";
+                    position: absolute;
+                    bottom: -1px;
+                    left: 0;
+                    width: 100%;
+                    height: 1px;
+                    background-color: $white;
+                }
+            }
+        }
```
Comment (kruchy8) [inline components/04-views/dashboard/default/_dashboard.scss]: Use vars within whole `__list` block

**Comment by kjugi** [comment]
In fact, it's not related to this task and I've created an issue to this here #276. We don't have too much time to repair all of this code right now 😕

File: components/03-modules/dashboard/nav/_nav.scss (hunk 1 above)
Comment (kjugi) [reply components/03-modules/dashboard/nav/_nav.scss]: it's related to magento theme and links in dashboard, we don't have here any classes on link it's just `a`

**Pull request merged**